src/test/regress/spec/isolation_shard_rebalancer_progress.spec

```
@@ -171,7 +171,8 @@ step "s7-get-progress"
 		targetport,
 		target_shard_size,
 		progress,
-		operation_type
+		operation_type,
+		source_lsn >= target_lsn AND source_lsn > '0/0' as lsn_sanity_check
```

I think this extra column would be good to add too:

```suggestion
		source_lsn >= target_lsn AND source_lsn > '0/0' as lsn_sanity_check,
		target_lsn = '0/0' as target_lsn_available
```